policyengine_us/reforms/tax_payroll_not_benefits/tax_payroll_not_benefits_reform.py

```diff
@@ -0,0 +1,94 @@
+from policyengine_us.model_api import *
+
+def create_tax_employer_payroll() -> Reform:
```

Comment:
You will also need to add this to the list of existing reforms - see other reform PRs